src/components/RichTextEditor/editor-config.ts

```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    createAlignPlugin,
+    createBoldPlugin,
+    createCodeBlockPlugin,
+    createCodePlugin,
+    createHeadingPlugin,
+    createItalicPlugin,
+    createLinkPlugin,
+    createListPlugin,
+    createParagraphPlugin,
+    createPlateUI,
+    createPlugins,
+    createSoftBreakPlugin,
+    createStrikethroughPlugin,
+    createUnderlinePlugin,
+    ELEMENT_LI,
+    ELEMENT_LIC,
+    ELEMENT_LINK,
+    ELEMENT_OL,
+    ELEMENT_UL,
+    LinkElement,
+    MARK_BOLD,
+    MARK_CODE,
+    MARK_ITALIC,
+    MARK_STRIKETHROUGH,
+    MARK_UNDERLINE,
+} from "@udecode/plate";
+import { ListItemElement } from "./components/elements/list-item";
```

Comment:
You could create an `index.ts` in the components directory where you export all components

Reply:
Done 💯